src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -39,9 +39,15 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
-  if (type === 'GROUP') {
+  if (type === CHANNEL_GROUP) {
     return (
-      <View style={[styles.containerImage, styles.containerDefaultImage, styles.containerDarkBlue]}>
+      <View
+        style={[
+          styles.containerImage,
+          styles.containerDefaultImage,
+          styles.containerDarkBlue,
+          style
+        ]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check for the type of channel has been updated to use the constant `CHANNEL_GROUP` instead of the hard-coded string 'GROUP'. Also, an additional style prop `style` has been added to the View component. This allows for more flexible styling of the component.
```diff
- if (type === 'GROUP') {
+ if (type === CHANNEL_GROUP) {
    return (
-     <View style={[styles.containerImage, styles.containerDefaultImage, styles.containerDarkBlue]}>
+     <View
+       style={[
+         styles.containerImage,
+         styles.containerDefaultImage,
+         styles.containerDarkBlue,
+         style
+       ]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->